.github/workflows/task-test.yml

```diff
@@ -57,6 +61,10 @@ jobs:
           sparse-checkout: |
             .install
             tests/pytests/requirements.*
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+            python-version: '3.10'
```

Comment:
Awesome.
We need to make sure this works before we install the dependencies (some OS don't have ANYTHING installed).
We can then remove the Python installation from the mariner script.

Reply:
https://github.com/RediSearch/RediSearch/actions/runs/7198170263/job/19608169830?pr=4194
It is running before the setup step and seems to work

as for mariner - 👍

Reply:
I’m mostly worry about amzn2. I would try to trigger manually the Linux platforms flow.
Also, these changes should be applied to the build artifacts flow as well

Reply:
good idea
I triggered the linux workflow